ReportSimulationOutput/measure.rb

```diff
@@ -353,6 +365,28 @@ def energyPlusOutputRequests(runner, user_arguments)
       if args[:include_timeseries_fuel_consumptions]
         result << OpenStudio::IdfObject.load("Output:Meter,ElectricStorage:ElectricityProduced,#{args[:timeseries_frequency]};").get
       end
+
+      # Resilience
+      if args[:include_annual_resilience] || args[:include_timeseries_resilience]
+        resilience_frequency = 'timestep'
```

Comment:
So for an HPXML w/ a battery, the default will now be to request timeseries data for multiple variables and then do some calculations on a timeseries basis. I want to make sure there is not a significant impact on runtime from this; if there is, we should probably default `include_annual_resilience` to false.

Reply:
Sample File | Timestep | Timeseries Freq | Completed simulation | Completed In
-- | -- | -- | -- | --
base-pv-battery.xml | 60 | Timestep | 3.5s | 7.6s
base-pv-battery.xml | 60 | Hourly | 3.4s | 7.1s
base-pv-battery.xml | 60 | Monthly | 3.1s | 6.5s
base-pv-battery.xml | 60 | Daily | 3.2s | 6.6s
base-pv-battery.xml | 10 | Timestep | 16.9s | 24.1s
base-pv-battery.xml | 10 | Hourly | 16.3s | 20.1s
base-pv-battery.xml | 10 | Monthly | 15.6s | 19.1s
base-pv-battery.xml | 10 | Daily | 16.1s | 19.6s